helm/alfresco-content-services/values.yaml

```
@@ -538,6 +551,87 @@ elasticsearch:
   enabled: true
   replicas: 1
   clusterHealthCheckParams: "wait_for_status=yellow&timeout=1s"
+elasticsearch-audit:
+  # -- Enables the embedded elasticsearch cluster for alfresco-audit-storage
+  enabled: true
+  nameOverride: elasticsearch-aas
+  replicas: 1
+  clusterHealthCheckParams: "wait_for_status=yellow&timeout=1s"
+  clusterName: elasticsearch-aas
+  extraEnvs:
+    - name: ELASTIC_USERNAME
+      valueFrom:
+        secretKeyRef: 
+          name: *aas_elasticsearch_secretName
+          key: AUDIT_ELASTICSEARCH_USERNAME
+    - name: ELASTIC_PASSWORD
+      valueFrom:
+        secretKeyRef: 
+          name: *aas_elasticsearch_secretName
+          key: AUDIT_ELASTICSEARCH_PASSWORD
+  ingress:
+    # -- toggle deploying elasticsearch-audit ingress for more details about configuration check
+    # https://github.com/elastic/helm-charts/blob/main/elasticsearch/values.yaml#L255
+    enabled: false
+alfresco-audit-storage:
+  enabled: true
+  image:
+    repository: quay.io/alfresco/alfresco-audit-storage
+    tag: 0.0.1-A8
+  messageBroker:
+    existingConfigMap:
+      # -- Name of the configmap which holds the message broker URL
+      name: *infrastructure_cmName
+    existingSecret:
+      # -- Name of the configmap which holds the message broker credentials
+      name: *acs_messageBroker_secretName
+  index:
+    existingConfigMap:
+      name: *infrastructure_cmName
+      keys:
+        url: AUDIT_ELASTICSEARCH_URL
+    existingSecret:
+      name: *aas_elasticsearch_secretName
+      keys:
+        username: AUDIT_ELASTICSEARCH_USERNAME
+        password: AUDIT_ELASTICSEARCH_PASSWORD
+kibana-audit:
+  enabled: true
+  healthCheckPath: "/kibana/app/kibana"
+  # -- Makes sure there is no default elasticsearch hosts defined
+  elasticsearchHosts: ""
+  # -- All of the values has to be set there to escape the issue with overriding the values
```

```suggestion
  # All of the values has to be set there to escape the issue with overriding the values
```

see if the readme improve - in the end this is not really a comment for the chart end user